src/languages/python-repl.js

```diff
@@ -0,0 +1,29 @@
+/*
+Language: Python REPL
+Requires: python.js
+Author: Josh Goebel <hello@joshgoebel.com>
+Category: common
+*/
+
+export default function(hljs) {
+  return {
+    aliases: ['pycon'],
+    contains: [
+      {
+        className: 'meta',
+        starts: {
+          // a space separates the REPL prefix from the actual code
+          // this is purely for cleaner HTML output
+          end: / |$/,
+          starts: {
+            end: '$', subLanguage: 'python'
+          }
+        },
+        variants: [
+          { begin: /^>>>(?=[ ]|$)/ },
+          { begin: /^\.\.\.(?=[ ])/ }
```

Comment:
```suggestion
          { begin: /^\.\.\.(?=[ ]|$)/ }
```

There can be an empty `...` line in REPL, no? (see my example)

Reply:
Oh how did I miss that.  These are edge cases though really. :)